src/sentry/static/sentry/app/components/alertLink.tsx

```diff
@@ -69,9 +67,23 @@ export default class AlertLink extends React.Component<Props> {
   }
 }
 
-const StyledLink = styled((props: StyledLinkProps) => (
-  <Link {...omit(props, ['withoutMarginBottom', 'priority', 'size'])} />
-))`
+export default AlertLink;
+
+const StyledLink = styled(({openInNewTab, to, href, ref, ...props}: StyledLinkProps) => {
+  const linkProps = omit(props, ['withoutMarginBottom', 'priority', 'size']);
+  if (href) {
+    return (
+      <ExternalLink
+        {...linkProps}
+        href={href}
+        ref={ref as React.RefObject<HTMLAnchorElement>}
```

Comment:
I don't think this is needed (or works as you expect)

Reply:
As we are now using `type AnchorProps = React.HTMLProps<HTMLAnchorElement>` I had some issues with the ref and I decided to do it like this for now ...